src/agent/src/netlink.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Kata Maintainers
+// Copyright (c) 2021 Apple Inc.
```

Comment:
I don't think this one is related to that PR.

Reply:
agreed, we should drop.